components/brave_wallet_ui/page/container.tsx

```diff
@@ -110,6 +110,9 @@ function Container (props: Props) {
   const [buyAmount, setBuyAmount] = React.useState('')
   const [sendAmount, setSendAmount] = React.useState('')
   const [selectedWidgetTab, setSelectedWidgetTab] = React.useState<BuySendSwapTypes>('buy')
+  const [lastCheckedTime, setLastCheckedTime] = React.useState<number>()
+  const [currentMousePosition, setCurrentMousePosition] = React.useState({ x: 0, y: 0 })
+  const [lastMousePosition, setLastMousePosition] = React.useState({ x: 0, y: 0 })
```

Comment:
Can we please move all this state and the associated `useMemo` / `useEffect` to a custom hook? This is not wallet-specific code so we could cut down the clutter a bit.

Reply:
Done!